tests/api/test_account.py

```diff
@@ -408,6 +452,180 @@ def test_customer_update(
     assert not data['user']['isActive']
 
 
+def test_logged_customer_update(user_api_client, address):
+    query = """
+    mutation UpdateCustomer($id: ID!, $billing: AddressInput,
+                            $shipping: AddressInput) {
+        loggedCustomerUpdate(id: $id,
+          input: {
+            defaultBillingAddress: $billing,
+            defaultShippingAddress: $shipping,
+        }) {
+            errors {
+                field
+                message
+            }
+            user {
+                id
+                email
+                defaultBillingAddress {
+                    id
+                }
+                defaultShippingAddress {
+                    id
+                }
+            }
+        }
+    }
+    """
+
+    # this test requires addresses to be set and checks whether new address
+    # instances weren't created, but the existing ones got updated
+    user = user_api_client.user
+    assert user.default_billing_address
+    assert user.default_shipping_address
+    billing_address_pk = user.default_billing_address.pk
+    shipping_address_pk = user.default_shipping_address.pk
+
+    id = graphene.Node.to_global_id('User', user.id)
+    address_data = convert_dict_keys_to_camel_case(address.as_data())
+
+    new_street_address = 'Updated street address'
+    address_data['streetAddress1'] = new_street_address
+
+    variables = {
+        'id': id, 'billing': address_data,
+        'shipping': address_data}
+    response = user_api_client.post_graphql(query, variables)
+    content = get_graphql_content(response)
+
+    User = get_user_model()
+    customer = User.objects.get(email=user.email)
```

Comment:
I don't understand this query, can't we just use `user`? Or refresh it from db if needed?